### PR TITLE
feature: Send proposal approved notification

### PR DESCRIPTION
**Pull request by alexdeveng**

### CODE REVIEW

File: src/main/java/com/acme/conferencesystem/cfp/proposals/business/ProposalService.java [hunk 1]
```diff
@@ -30,8 +41,10 @@ public List<Proposal> getAllProposals() {
     }
 
     public List<Proposal> getAcceptedProposals() {
-        return repository.getProposalEntityByStatus(ProposalStatus.ACCEPTED).stream()
+        return repository.getProposalEntityByStatus(ProposalStatus.ACCEPTED)
+                .stream()
                 .map(mapper::entityToProposal)
+                .peek(p -> LOG.info(p.toString()))
```
Comment (anyulled) [inline src/main/java/com/acme/conferencesystem/cfp/proposals/business/ProposalService.java:47]: What's the point of this?

File: src/test/java/com/acme/conferencesystem/ContainerConfig.java [hunk 2]
```diff
@@ -13,7 +13,8 @@ public class ContainerConfig {
     @ServiceConnection
     @RestartScope
     public PostgreSQLContainer<?> postgreSQLContainer() {
-        return new PostgreSQLContainer<>("postgres:16-alpine");
+        return new PostgreSQLContainer<>("postgres:16-alpine")
+                .withCommand("postgres", "-c", "max_connections=200");
```
Comment (anyulled) [inline src/test/java/com/acme/conferencesystem/ContainerConfig.java:17]: what for?

File: src/test/java/com/acme/conferencesystem/cfp/talks/http/TalkControllerIntegrationTest.java [hunk 3]
```diff
@@ -36,31 +35,16 @@ void retrieveTalks() {
         ProposalTestAPI.createAcceptedProposal(speakerId2, requestSpecification);
         ProposalTestAPI.createAcceptedProposal(speakerId2, requestSpecification);
         ProposalTestAPI.createProposal(speakerId2, requestSpecification);
-
```
Comment (anyulled) [inline src/test/java/com/acme/conferencesystem/cfp/talks/http/TalkControllerIntegrationTest.java:39]: could we remove this commented code?

**Review by anyulled — APPROVED** [review]

File: src/test/java/com/acme/conferencesystem/ContainerConfig.java (hunk 2 above)
Comment (alexdeveng) [reply src/test/java/com/acme/conferencesystem/ContainerConfig.java:17]: There is an issue with the connections, I think with flyway. I don't have capacity right now to have a look, so it was a work arround xD

**Pull request merged**